modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -53,52 +53,62 @@ $ vim bmh.yaml
 +
 [source,yaml]
 ----
----
 apiVersion: v1
-kind: Secret
-metadata:
-  name: openshift-worker-<num>-bmc-secret <1>
-type: Opaque
-data:
-  username: <base64-of-uid> <2>
-  password: <base64-of-pwd> <3>
----
-apiVersion: metal3.io/v1alpha1
-kind: BareMetalHost
-metadata:
-  name: openshift-worker-<num> <1>
-spec:
-  online: true
-  bootMACAddress: <NIC1-mac-address> <4>
-  bmc:
-    address: <protocol>://<bmc-ip> <5>
-    credentialsName: openshift-worker-<num>-bmc-secret <1>
-  networkConfig: <6>
-    interfaces:
-    - name: <NIC1_name>
-      type: ethernet
+ kind: Secret
```

Comment:
It is important to note that the network configuration is optional. The secret and the reference to it only need to be added if the user wants to add network settings to the newly created node.

Reply:
Revised the [callout entry](https://github.com/openshift/openshift-docs/pull/45918/files#diff-f90b46cde12693fcb811ca202b3759cea7652922c2b7f92da22716e5e8aaa180R105).